web_src/js/features/repo-diff.js

```diff
@@ -178,19 +189,22 @@ function initRepoDiffShowMore() {
     $target.addClass('disabled');
 
     const url = $target.data('href');
-    $.ajax({
-      type: 'GET',
-      url,
-    }).done((resp) => {
+
+    try {
+      const response = await GET(url);
+      const resp = await response.text();
+
       if (!resp) {
         $target.removeClass('disabled');
         return;
       }
```

Comment:
```suggestion
```

Reply:
Why?

Reply:
Because finally has done that.

Reply:
Maybe remove only `$target.removeClass('disabled');`? To not continue of `resp` is falsy

Reply:
Done https://github.com/go-gitea/gitea/pull/29743/commits/f7691aebc0a6e47fee0e02bcd639b2e87019d8e1